test/js/bun/net/server.tests.ts

```diff
@@ -0,0 +1,450 @@
+import { describe, expect, it } from "bun:test";
+import { close } from "fs";
+import { createServer } from "net";
+import { createCallCheckCtx } from "node-test-helpers";
```

Comment:
```suggestion
import { createCallCheckCtx } from "node-harness";
```

Reply:
Thanks done in https://github.com/oven-sh/bun/pull/2337/commits/62f4f55db5fd5d7677124fe3cbb0bffd5f5e9366 when moved the test